raiden/tests/integration/long_running/test_token_networks.py

```diff
@@ -49,6 +48,7 @@ def is_manager_saturated(connection_manager, registry_address, token_address):
         lambda channel_state:
         (
             is_channel_open_and_funded(channel_state) and
+            channel_state.partner_state.address != connection_manager.BOOTSTRAP_ADDR and
```

Comment:
could you please move this to a normal function? I would really prefer if we stick to single line lambdas